src/core/Table/TableRowMemoized.tsx

```diff
@@ -70,6 +79,38 @@ const TableRow = <T extends Record<string, unknown>>(props: {
 
   const refs = useMergedRefs(rowRef, mergedProps.ref);
 
+  const subRowExpanderCellIndex = row.cells.findIndex(
+    (c) => c.column.id !== SELECTION_CELL_ID,
+  );
+
+  const getSubRowExpander = (cell: Cell<T>) =>
+    expanderCell ? (
+      expanderCell({
+        ...tableInstance,
```

Comment:
hmm.. I wonder if we could just apply tableInstance props in expanderCell function in `Table`, so we dont need to pass whole instance to the memoized row.

Reply:
But I still need to pass `cell` which I don't have outside of row.

Reply:
Yeah, but you would avoid passing whole table instance.. Just would be not nice to add it in table as this is prop passed by the user...

Reply:
so instead of passing `instance` and verbatim `expanderCell` to `TableRow`, you would pass a wrapper around `expanderCell`? The wrapper would apply `instance` from Table?

Reply:
Yeah, that was the idea, but not sure it is better now :(